xapian-applications/omega/mimemap.tokens

```diff
@@ -214,6 +214,23 @@ tif		image/tiff
 tiff		image/tiff
 webp		image/webp
 
+# Audio files:
+flac		audio/flac
+mp3		audio/mpeg
+ogg		audio/ogg
```

Comment:
There's also `oga` and `spx` for `audio/ogg`, and `ogv` for `video/ogg` - see https://tools.ietf.org/html/rfc5334#section-10.2

I think we probably don't want an extension to mime-type mapping for the `ogg` extension itself though - `ogg` can be `audio/ogg` or `video/ogg` (as your example files demonstrate!)  If we don't provide a mapping then omindex will use libmagic which can tell the difference from the file content:

```
$ file --mime-type testfiles/*/*.ogg
testfiles/audio/file_example_OOG_1MG.ogg:       audio/ogg
testfiles/video/file_example_OGG_480_1_7mg.ogg: video/ogg
```

(`file` uses libmagic internally.)

Both get mapped to libextractor by default, but the recorded mime-type will be wrong (which could matter if that's provided as a "file type" filter to users) and it'll also be confusing to the user if they overrides the default mappings.

Reply:
Thank you for the insight, I have changed this.

Reply:
You've addressed the second part, but not the missing extensions I noted in the first paragraph.

We should pick up the type of such files via libmagic, but it's good to provide extension to mime type mappings where they're reliable since libmagic needs to look at the start of the file to determine the type, which takes more time than just looking at the end of the filename - not much per file, but it can add up.

Reply:
Oh, I misunderstood the comment. I will fix this and update the PR.

Reply:
I have fixed this